public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -153,9 +164,14 @@ class UnConnectedExploreToolbar extends PureComponent<Props> {
                 Split
               </ToolbarButton>
             ) : (
-              <ToolbarButton tooltip="Close split pane" onClick={() => closeSplit(exploreId)} icon="times">
-                Close
-              </ToolbarButton>
+              <>
+                <ToolbarButton title="Resize pane" icon="columns" disabled={isLive} onClick={onClickResize}>
```

Comment:
The icon looks quite off compared to other icons in the toolbar, also the tooltip doesn't tell what type of resize it is. Could we use simpler icons that would represent maximizing/minimizing? At the moment it looks more like it represents adding or removing panes.

<img width="229" alt="Screen Shot 2022-09-12 at 14 01 26" src="https://user-images.githubusercontent.com/745532/189648442-79e9a1f2-a8a6-459e-96cd-2cb5e8e38a40.png">

Reply:
This is submitted to tomorrow's UX feedback session to discuss the icons and tooltip.

Reply:
After some discussion and looking at what icons we have available, I think a simple > and < on this portrays the action nicely. I also changed the tooltip to be more informative. Let me know what you think!